CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- Energy Yield Analysis (EYA) to Operational Assessment(OA) Gap Analysis method
+- Uncertainty quantification
```

Comment:
UQ has always been a part of AEP analysis. Do you mean uncertainty quantification for electrical losses and longterm turbine gross energy (Nicola's recent PR)? This is still in develop though. Soon we can also add "Augmented AEP method with variable temporal resolution, machine learning methods, and improved uncertainty quantification" too, but the PR is still open.